### ISSUE TITLE
[Feature Request]Special subscribe

### ISSUE THREAD

**Issue by NE-SmallTown** [isaacs/github#928]
At now,if we want to know something of a repo,we can watch it,and then every issue update will notify us.But we are not interested in all of them,probably just some of them.But we can't distinguish the interesting and not interesting in the homepage notifications.

So,I think maybe we can add a "Special Subscribe"(or other name) button below the "subscribe" button.And then mark a label when notify them in the homepage or add a filter option which we can choose.

**Comment by NE-SmallTown** [comment]
I am not sure the name,maybe ":special-subscribe",":voluntary-subscribe"

**Comment by refack** [comment]
by label

**Comment by NE-SmallTown** [comment]
@refack I am not sure what you mean,but label can't do this

**Comment by refack** [comment]
@NE-SmallTown, I know. But I wish it did... (Subscribe to issues/PRs by label)

**Comment by TPS** [comment]
Kinda a version of https://github.com/isaacs/github/issues/410#issuecomment-118060114

**Comment by NE-SmallTown** [comment]
@TPS thanks,but I am not talk about releases,I don't want to cancel subscribe of issues,I just want to enhance subscribe of some specified issues and then I can see them in the issue panel.

**Comment by TPS** [comment]
@NE-SmallTown That's what I requested, too:
>> As I mentioned, it'd be amazing to subscribe receive notifications of _any_ page of a projec.

Even a `label:` search is included in "_any_ page." 😁

**Comment by refack** [comment]
P.S. I use https://www.changedetection.com/ to track changes to pages.

[@gitbug this doesn't get you off the hook as it's a low-quality work around 😜]

**Comment by NE-SmallTown** [comment]
@refack thanks for your share,but I have associate github with my email so I will receive all notifications when a issue/pr etc. update.I want check the special change in my issue panel,that's very make sense.

**Comment by apphead** [comment]
Please somebody @github address this minor feature enhancement. This suggestion looks sound to me:
https://github.com/isaacs/github/issues/410#issuecomment-291797759

**Comment by NE-SmallTown** [comment]
close for no response

**Comment by TPS** [comment]
@NE-SmallTown We should close almost this whole repo for no response,  but that isn't really its purpose, right?

**Comment by refack** [comment]
> @NE-SmallTown We should close almost this whole repo for no response, but that isn't really its purpose, right?

Apropo, we need a 😞 reaction ( 😕 doesn't cover enough cases)

**Comment by NE-SmallTown** [comment]
ahh, I agree, so what's your thought about this? reopen?

**Comment by jcrben** [comment]
Sounds this like might be paired with an ability for the repo owners to emit a special sort of notification...?

**Comment by NE-SmallTown** [comment]
No, not repo owner, the feature what I want is for anyone who want to subscribe
